feat(file_sharing): Provide template creator list in public shares

artonge opened the pull request:
Fix https://github.com/nextcloud/server/issues/48064

artonge commented:
/backport to stable30

artonge commented:
/compile rebase

artonge commented:
@camilasan @elzody just pushed an update, to also handle the file creation. Can you test it?

elzody reviewed (approved):
Gave it a try, it seems to work fine on my end.

artonge committed:
fix(encryption): Correctly set encrypted to 0 when copying

If encryption got disabled, copying should set encrypted to 0 for the
 new unencrypted copy. For instance when using encryption:decrypt-all

Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>
Signed-off-by: Louis Chemineau <louis@chmn.me>

nextcloud-command committed:
chore(assets): Recompile assets

Signed-off-by: nextcloud-command <nextcloud-command@users.noreply.github.com>

artonge commented:
/backport [6d672c4](https://github.com/nextcloud/server/pull/54565/commits/6d672c4cba5624e6c35411f321110ea2895db0f0) to stable32

artonge commented:
/backport [6d672c4](https://github.com/nextcloud/server/pull/54565/commits/6d672c4cba5624e6c35411f321110ea2895db0f0) to stable31

artonge commented:
/backport 6d672c4c to stable31
